deepchem/feat/bio_seq_featurizer.py

```diff
@@ -325,4 +329,48 @@ def _featurize(self, datapoint):
 
         datapoint.close()
 
-        return np.array(features, dtype="object")
\ No newline at end of file
+        return np.array(features, dtype="object")
+
+
+class FASTAFeaturizer(Featurizer):
+    """
+        Featurizes FASTA files by extracting the sequence names and sequences.
+        Each sequence in the FASTA file is represented as a list containing the
```

Comment:
Explain the format of the featurized output. What is its shape? What is its type?